ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -218,18 +220,22 @@ private async Task<IEnumerable<IList<Notification>>> GetGroupedResultsAsNotifica
             return notificationsToReturn;
         }
 
-        private List<TreatmentEvent> CombineTreatmentEvents(Notification notification,
+        private async Task<List<TreatmentEvent>> CombineTreatmentEvents(Notification notification,
             MigrationDbNotification rawNotification,
             IEnumerable<TreatmentEvent> notificationTransferEvents,
-            IEnumerable<TreatmentEvent> notificationOutcomeEvents)
+            IEnumerable<TreatmentEvent> notificationOutcomeEvents,
+            PerformContext context,
+            int runId)
         {
             return notification.ClinicalDetails.IsPostMortem == true
                 ? TreatmentEventsForPostMortemNotification(rawNotification, notificationOutcomeEvents)
```

Comment:
When we add the diagnosis event to post mortem notifications, we'll also have to add the notifying info to that